packages/testcontainers/src/containers/RegTestContainer/Masternode.ts

```diff
@@ -17,7 +22,7 @@ export class MasterNodeRegTestContainer extends RegTestContainer {
    */
   constructor (masternodeKey: MasterNodeKey = RegTestFoundationKeys[0], image: string = DeFiDContainer.image, options?: DockerOptions) {
```

Comment:
```suggestion
  constructor (private readonly masternodeKey: MasterNodeKey = RegTestFoundationKeys[0], image: string = DeFiDContainer.image, options?: DockerOptions) {
```

How about this so that we can remove `_masternodeKey` and `get masternodeKey (): MasterNodeKey {`?